Override default to match what other debian derivatives do.

Rather than having lib being a link of lib64, do the the contrary:
use lib and make lib64 a link of it.

--- gcc/config/i386/t-linux64.orig	2008-10-22 03:33:48.513859286 +0200
+++ gcc/config/i386/t-linux64	2008-10-22 03:34:27.928844912 +0200
@@ -13,7 +13,7 @@
 
 MULTILIB_OPTIONS = m64/m32
 MULTILIB_DIRNAMES = 64 32 
-MULTILIB_OSDIRNAMES = ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
+MULTILIB_OSDIRNAMES = ../lib $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib
